chaincode-labs/chaincode-podcast/privacy-and-robustness-in-lightning.md

```diff
@@ -331,112 +351,111 @@ When it's my turn, by definition, you haven't got anything in flight.
 So I can say, hey, it's time to upgrade.
 And it's much simpler in that way too.
 So this is something that we just learned through experience that we over-designed the original protocol.
-And now whether this becomes a separate change or goes in with an L2 change or not, this is something we're going to discuss at the summit.
+And now whether this becomes a separate change or goes in with an  Eltoo change or not, this is something we're going to discuss at the summit.
 But from my point of view, we end up with a more robust and simple protocol at the end of this.
 And it turns out that supporting both is actually pretty easy.
 
-Speaker 2: 00:16:10
+Adam Jonas: 00:16:10
 
-So you're talking about when we were talking about L2, you're talking about changes and you're sort of casually mentioning the soft fork piece of it.
-I mean, there's not going well, there's Greg Sanders will be there and he's obviously thinking about this, but you don't really have the people at the table that are actually doing like would be pulling the soft fork together that are that really understand the lightning case, I guess.
-Maybe AJ, maybe InstaGibs.
+When we were talking about  Eltoo, you're talking about changes and you're sort of casually mentioning the soft fork piece of it.
+I mean, there's not going well, there's Greg Sanders will be there and he's obviously thinking about this, but you don't really have the people at the table that are actually doing like would be pulling the soft fork together that really understand the lightning case, I guess.
+Maybe AJ, maybe Instagibbs.
 The list gets very short after that.
 So, like, how are you able to communicate those needs down to the lower layer?
 
-Speaker 0: 00:16:47
+Rusty Russell: 00:16:47
 
 Well, I think the any prep out cases have been pretty strong for a long time.
 The question's always been, do we want other soft fork as well?
 
 ## CTV
 
-Speaker 0: 00:16:54
+Rusty Russell: 00:16:54
 
-And people are like, oh, if we had obstacle the check sick from stack, we can also do it.
+And people are like, oh, if we had obstacle the `CHECKSIG` from stack, we can also do it.
 Sure.
 I can probably build a car out of matchsticks and snot, but I don't want to.
 And don't make me do it.
 
-Speaker 2: 00:17:09
+Adam Jonas: 00:17:09
 
 Could you do that?
 
-Speaker 0: 00:17:12
+Rusty Russell: 00:17:12
 
 A theoretical engineer probably could do that.
 I could not do that.
 I'm not crafty at all.
 But, okay.
 That is probably an aside.
 We don't need to go down.
-But, you know, check sync from stack is like, oh, it's cool that you can even make that work.
+But, you know, `CHECKSIG` from stack is like, it's cool that you can even make that work.
```

Comment:
This actually refers to the opcode `OP_CHECKSIGFROMSTACK`.
Make sure to also change it in the rest of the transcript.